tests/data/MerchFactory.ts

```diff
@@ -86,6 +86,7 @@ export class MerchFactory {
       description: faker.lorem.sentences(2),
       start,
       end,
+      orders: [],
```

Comment:
The issue's that if it's excluded, then the `toEqual` check in our test won't pass since every time `getFuturePickupEvents` is called, then `orders: []` will be set. But the 'expected' models don't have `orders: []` if this line is removed.

Reply:
👍🏽